config/deployment.yml

```diff
@@ -43,19 +43,25 @@ spec:
         volumeMounts:
         - name: template-fs
           mountPath: /etc/kappctrl-mem-tmp
-        securityContext:
-          runAsUser: 1000
-          runAsGroup: 2000
+        - name: home
```

Comment:
do we use home directory for anything?

Reply:
Yeah, though it basically is a temp directory as it stands now, just mounted at `/home/kapp-controller`. I think we want to avoid writing to the rootfs and so we'll need a scratch volume to write these things to (unless they can just be held in-memory).